testing/tests/DevExpress.localization/localization.globalize.tests.js

```diff
@@ -22,11 +22,11 @@ SystemJS.config({
 });
 
 define(function(require, exports, module) {
-    require('../../helpers/l10n/cldrNumberDataRu.js');
-    require('../../helpers/l10n/cldrNumberDataDe.js');
-    require('../../helpers/l10n/cldrCalendarDataRu.js');
-    require('../../helpers/l10n/cldrCurrencyDataRu.js');
-    require('../../helpers/l10n/cldrCurrencyDataEn.js');
+    const cldrData = [
```

Comment:
Can we use [ru.js](https://unpkg.com/browse/devextreme-cldr-data@1.0.2/ru.js) instead?